planning/behavior_velocity_planner/src/scene_module/intersection/util.cpp

```diff
@@ -74,6 +74,29 @@ bool hasLaneId(const autoware_auto_planning_msgs::msg::PathPointWithLaneId & p,
   return false;
 }
 
+std::optional<std::pair<int, int>> findLaneIdInterval(
```

Comment:
I think it's better to use size_t.
Is there any reason to use int instead of size_t?

Reply:
That's right. I'll replace index variables to size_t type.

Reply:
@tkimura4 Just fixed now!